core/geotime/src/main/java/org/locationtech/geowave/core/geotime/adapter/LatLonFieldMapper.java

```diff
@@ -46,8 +46,10 @@ public void initFromOptions(
     }
     xAxisFirst =
         inputFieldDescriptors.get(0).indexHints().contains(SpatialField.LONGITUDE_DIMENSION_HINT)
-            || !inputFieldDescriptors.get(1).indexHints().contains(
-                SpatialField.LONGITUDE_DIMENSION_HINT);
+            || inputFieldDescriptors.get(1).indexHints().contains(
+                SpatialField.LATITUDE_DIMENSION_HINT)
+            || inputFieldDescriptors.get(0).fieldName().equalsIgnoreCase("longitude")
```

Comment:
do we use inference on field names without hints in the case of an un-annotated POJO?  I had assumed with an unannotated POJO you're stuck with default index behaviors as if there are no index hints, but in looking at this addition of checking for longitude field names am curious the reasoning if its not automatically inferring index mapping based on name?

Reply:
Currently, index mappings are only inferred based on type.  Logic could be added to the mapping code to pass all compatible fields to the mapper and let the mapper pick appropriate fields, but that is not currently in place.